Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN mix phx.digest
 RUN mix compile
 RUN mix distillery.release
 
+RUN mix ecto.setup
```

Comment:
This won't work, because it will do the migration only when the image will be build, and not when it was deployed.
So the migration will be run on the local PostgreSQL instance (probably your dev instance) instead of the instance in the server.

```suggestion
```